node/sc/bridge_manager.go

```diff
@@ -312,7 +312,7 @@ func (bi *BridgeInfo) handleRequestValueTransferEvent(ev *RequestValueTransferEv
 
 		uri, err = erc721.TokenURI(nil, ev.ValueOrTokenId)
 		if err != nil {
-			if err == vm.ErrExecutionReverted {
+			if err.Error() == vm.ErrExecutionReverted.Error() {
```

Comment:
LGTM :)

Later on, it would be nice to find out why errors.Is() is not working.